src/WPF/ArcGISRuntime.WPF.Viewer/Samples/Data/QueryFeaturesWithArcadeExpression/QueryFeaturesWithArcadeExpression.xaml.cs

```diff
@@ -0,0 +1,148 @@
+        private const string RPDBeatsLayerName = "RPD Beats  - City_Beats_Border_1128-4500";
+
+        // Hold a reference to the callout text content to store it between clicks.
+        private string _calloutText = string.Empty;
+
+        public QueryFeaturesWithArcadeExpression()
+        {
+            InitializeComponent();
+            _ = Initialize();
+        }
+
+        private async Task Initialize()
+        {
+            try
+            {
+                // Create an ArcGIS portal item.
+                var portal = await ArcGISPortal.CreateAsync();
+                var item = await PortalItem.CreateAsync(portal, "14562fced3474190b52d315bc19127f6");
+
+                // Create a map using the portal item.
+                MyMapView.Map = new Map(item);
+            }
+            catch (Exception e)
+            {
+                MessageBox.Show(e.Message, "Error");
+            }
+
+            // Load the map.
+            await MyMapView.Map.LoadAsync();
+
+            // Set the visibility of all but the RDT Beats layer to false to prevent UI clutter.
+            MyMapView.Map.OperationalLayers.ToList().ForEach(l => l.IsVisible = l.Name == RPDBeatsLayerName);
+
+            // Hold the layer value so that the clicked geoelement can be identified in the event handler.
+            _layer = MyMapView.Map.OperationalLayers.FirstOrDefault(l => l.Name == RPDBeatsLayerName);
+        }
+
+        private async Task GeoViewTappedTask(GeoViewInputEventArgs e)
+        {
+            try
+            {
+                // Get the layer based on the position tapped on the MapView.
+                IdentifyLayerResult identifyResult = await MyMapView.IdentifyLayerAsync(_layer, e.Position, 12, false);
+
+                if (identifyResult == null || !identifyResult.GeoElements.Any())
+                {
+                    MyMapView.DismissCallout();
+
+                    return;
+                }
+
+                // Get the tapped GeoElement as an ArcGISFeature.
+                GeoElement element = identifyResult.GeoElements.First();
+                ArcGISFeature feature = element as ArcGISFeature;
+
+                // If the previously clicked feature is null or the previous feature ID does not match the current feature ID
+                // run the arcade expression query to get the crime count for a given feature.
+                if (_previousFeature == null || !(feature.Attributes["ID"].Equals(_previousFeature.Attributes["ID"])))
+                {
+                    // Show the loading indicator as the arcade evaluator evaluation call can take time to complete.
+                    MyLoadingGrid.Visibility = Visibility.Visible;
+
+                    // Instantiate a string containing the arcade expression.
+                    string expressionValue = "var crimes = FeatureSetByName($map, 'Crime in the last 60 days');\n" +
+                                          "return Count(Intersects($feature, crimes));";
```

Comment:
```suggestion
                                             "return Count(Intersects($feature, crimes));";
```
Might as well line it up.